CHANGELOG

```diff
@@ -35,8 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Updated
 
-- Bump DS version to 3.4.0
-- Bump nvmrc version to Node 20.
+- Bump nvmrc version to Node 20 and DS package to release candidate 3.4.0-rc.
 - Update error message in Item Filter Year to include new error message prefix from the DS.
```

Comment:
Revert this so that it does say `3.4.0` and the node 20 update as separate lines and not have the `-rc` indicated here.